ort-sys/build.rs

```diff
@@ -376,6 +376,19 @@ fn prepare_libort_dir() -> (PathBuf, bool) {
 	}
 }
 
+fn try_setup_with_pkg_config() -> bool {
+	match pkg_config::Config::new().probe("libonnxruntime") {
+		Ok(lib) => {
```

Comment:
Shouldn't this be using at least `lib.link_paths` here? FWICT, `probe` seems to only return the package configuration, it doesn't configure the link args

Reply:
Thanks for the hint, you are right. Fixed it.